.circleci/config.yml

```diff
@@ -452,6 +452,27 @@ jobs:
           no_output_timeout: 15m
       - magma_slack_notify
 
+  # Fail if code doesn't pass formatting requirements.
+  fmt:
+    machine:
+      image: ubuntu-1604:201903-01
+    environment:
+      MAGMA_ROOT: /home/circleci/project
+    steps:
+      - checkout
+      - build/determinator:
+          <<: *all_gateways_build_verify
+      - run:
+          # Ref: https://circleci.com/blog/enforce-build-standards/
+          name: "go fmt"
+          command: |
+            cd ${MAGMA_ROOT}
+            go fmt ./... > gofmt.log 2>&1
```

Comment:
Do you want to use `go imports` instead ? it might also fix the unused imports. Might be good to add these in some orc8r developer documentation to integrate goimports or gofmt with ide.